src/server.c

```diff
@@ -122,6 +140,10 @@ void get_file(int fd, struct cache *cache, char *request_path)
     ///////////////////
     // IMPLEMENT ME! //
     ///////////////////
+    // char file_path[5000];
+
+    printf("%d, %s, %d\n", fd, request_path, cache->cur_size);
```

Comment:
```suggestion
    printf("%d, %s\n", fd, request_path);
```
remove the `cache` for now, this seems to be causing the seg fault

Reply:
also add `resp_404(fd);` to this function for now.  so you can something output in your browser.